test-pom/pom.xml

```diff
@@ -54,7 +54,9 @@ THE SOFTWARE.
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>jenkins-test-harness</artifactId>
-      <version>2.41.1</version>
+<!--      <version>2.41.1</version>-->
```

Comment:
Are these versions commented on purpose?

Reply:
I am still correcting the tests, so it's WiP

Reply:
Ho actually I thought I pushed the 2.46 after the failing try with 2.42. Thx for the headsup!

Reply:
I think this bump should actually be reverted in not required for the scope of that PR. This seems like something that can (and if can, then should) be handled in a dedicated PR for clarity (and more speedy reviews because less code, good for everyone).

Reply:
This has been marked as ready for merge but the code is still commented. Shouldn't we just remove it?